code/mob/living/critter/flock/flockdrone.dm

```diff
@@ -775,7 +775,7 @@
 			return
 
 	// CONVERT TURF
-	if(!isturf(target) && !HAS_ATOM_PROPERTY(target,PROP_ATOM_FLOCK_THING))
+	if(!isturf(target) && !HAS_ATOM_PROPERTY(target,PROP_ATOM_FLOCK_THING) && !istype(target, /obj/flock_structure) && !istype(target, /obj/structure/girder))
```

Comment:
Girders are just gross but /obj/flock_structure should definitely have PROP_ATOM_FLOCK_THING. Possibly in its constructor. Possibly set when flockmind creates it.